ValidateRomResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace MMRando
```

Comment:
I know, subfolders are a huge pain to work with - let's just put random ass stuff in the root folder. Structure is overrated anyway.

That being said, could you move this to models folder? 😏

Reply:
I'd rather leave it in root over putting it into a completely unrelated folder where it can get lost

Reply:
Don't you just hate it when your files just get completely lost...

... in a subfolder